Revert/Sources/AppDelegate.swift

```diff
@@ -8,12 +8,10 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
   func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-    if floor(NSFoundationVersionNumber) > floor(NSFoundationVersionNumber_iOS_7_0) || UIDevice.currentDevice().userInterfaceIdiom == .Pad {
-      guard let splitViewController = self.window?.rootViewController as? UISplitViewController else {
-        fatalError("Root view controller should be a `UISplitViewController` on any device > iOS 7 or on iPads")
-      }
-      self.splitViewControllerDelegate.configureSplitViewController(splitViewController)
+    guard let splitViewController = self.window?.rootViewController as? UISplitViewController else {
```

Comment:
Would it be better to remove the whole `guard` then?

I just removed the check for old versions of iOS on this one, but this is the same check that was being done before. I'm not really sure what is the reason behind it. @cuva?